### ISSUE TITLE
Link buttons do not share the same style overrides as hyperlinks

### ISSUE THREAD

**Issue by Blackbaud-PaulCrowder** [blackbaud/skyux-toast#25]
### Expected behavior

`button` elements with a CSS class of `sky-btn-link` or `sky-btn-link-inline` should display with the same color and decoration (underline) overrides as anchor (`a`) elements.

### Actual behavior

`button` elements with a CSS class of `sky-btn-link` or `sky-btn-link-inline` display with their default styles.

### Steps to reproduce

- Create a custom toast component with the following markup:

```
<a href="https://example.com">Hyperlink</a>
<button type="button" class="sky-btn sky-btn-link-inline">Inline button</button>
```

- Notice that the link button does not have the darker text and underline style that the hyperlink does.
